fdm-app/app/components/custom/dropzone.tsx

```
@@ -0,0 +1,240 @@
+                    ref={inputRef}
+                    placeholder=""
+                    className="hidden"
+                    accept={
+                        typeof accept === "string" ? accept : accept?.join(",")
+                    }
+                    multiple={multiple}
+                    required={required}
+                    disabled={disabled}
+                />
+                <label
+                    tabIndex={0}
```

Biome (2.1.2)

[error] The HTML element label is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)